packages/theming/atlas/src/native/ts/core/widgets/textarea.ts

```diff
@@ -56,16 +59,25 @@ export const TextArea: TextBoxType = {
 };
 export const TextAreaVertical: TextBoxType = {
     container: TextBoxVertical.container,
+    containerDisabled: TextBoxVertical.containerDisabled,
     label: {
         ...TextBoxVertical.label,
         height: undefined,
         paddingVertical: undefined,
         textAlignVertical: undefined
     },
+    labelDisabled: {
+        ...TextBoxVertical.labelDisabled,
+        height: undefined,
```

Comment:
What's the purpose in explicitly setting height to undefined?

Reply:
This is one of the downsides of merging `TextBox` & `TextBoxVertical`. Because we set the height in `TextBox.labelDisabled`, we have to unset it here, because we don't need it when vertical.

Reply:
I see, thanks. ✅